lib/worker.js

```diff
@@ -15,15 +15,18 @@ const eslintPathLocal = Path.join(FS.realpathSync(Path.join(__dirname, '..')), '
 let eslint = null
 let prefixPath = null
 
-function find(startDir, name) {
-  let filePath
+function find(startDir, names) {
```

Comment:
Updated to test for type `string`.
Added a new `localNames` variable to avoid param reassign.